platforms/corda-ent/configuration/roles/helm_component/templates/nmap.tpl

```diff
@@ -0,0 +1,65 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: {{ component_name }}
+  namespace: {{ component_ns }}
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: {{ component_name }}
+  chart:
+    git: {{ git_url }}
+    ref: {{ git_branch }}
+    path: {{ charts_dir }}/nmap
+  values:
+    nodeName: {{ node_name }}
+    metadata:
+      namespace: {{ component_ns }}
+    notaryName: {{ notary_name }}
+    storage:
+      name: {{ storageclass }}
+    replicas: 1
+    image:
+      initContainerName: {{ init_container_name }}
+      imagepullsecret: {{ image_pull_secret }}
+    dockerImage:
+      name: {{ nmap_image_name }}
+      tag: {{ nmap_image_tag }}
+      pullPolicy: Always
+      imagePullSecret: {{ image_pull_secret }}
+    acceptLicense: YES
+    vault:
+      address: {{ vault_addr }}
+      certsecretprefix: {{ vault_cert_secret_prefix }}
+    ambassador:
+      external_url_suffix: {{ external_url_suffix }}
+    volume:
+      baseDir: /opt/corda
+    service:
+      port: {{ nmap_port.servicePort }}
+    serviceInternal:
+      port: 5050
+    shell:
+      sshdPort: 2222
+      user: {{ ssh_username }}
+      password: {{ ssh_password }}
+    database:
+      driverClassName: "org.h2.Driver"
+      url: "jdbc:h2:file:./h2/networkmap-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
+      user: {{ db_username }}
+      password: {{ db_password }}
+      runMigration: "true"
+    identityManager:
+      host: idman.{{ component_ns }}
```

Comment:
Fetch 'idman' from network.yaml

Reply:
Also add the ambassador notations for nmap.

Reply:
This is fixed